lib/simulator-management.js

```diff
@@ -216,11 +217,11 @@ async function shutdownOtherSimulators (currentDevice) {
 
 async function getInstalledBundleIds (device, candidateBundleIds) {
   const bundleIds = [];
-  for (const bundleId of candidateBundleIds) {
+  await B.filter(candidateBundleIds, async (bundleId) => {
     if (await device.isAppInstalled(bundleId)) {
```

Comment:
not really - the filter condition should be `async  (bundleId) => await device.isAppInstalled(bundleId)`

Reply:
and the result could be returned immediately. There is no need for the intermediate accumulator